src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PathSeqScoreSpark.java

```diff
@@ -136,13 +137,15 @@
 
     private int recommendedNumReducers = 0;
 
+    //TODO: fix this
```

Comment:
looks like the same thread as above, I assume the todos mean we should hit them later instead of in this PR?  Should we open an issue?

Reply:
Replaced these with https://github.com/broadinstitute/gatk/issues/6610 to track remaining todos.